README.md

```diff
@@ -29,6 +29,10 @@ To locally serve the site at [localhost:8888][], run the following command:
 $ npm run serve
 ```
 
+> **WARNING:** If you want to **check links** locally, you (temporarily) might have
```

Comment:
I'm not sure this should be a warning -- the build won't break if the link checker doesn't run locally. Maybe `Note`, or `Caution`?

Reply:
Good point. I went with **NOTE**.